<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlatformServerGUI</class>
 <widget class="QMainWindow" name="PlatformServerGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Server Details</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>*You can find a log file of all the users' info at /home/[userName]/Desktop/log*****.txt</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="PortvalLabel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="IPvalLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Server is running on IP:</string>
     </property>
    </widget>
    <widget class="QLabel" name="PortLabel">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="statustextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>371</width>
       <height>261</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quitButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>350</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>390</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <action name="actionServer_Log">
   <property name="text">
    <string>Server Log</string>
   </property>
  </action>
  <action name="actionBooks_Database">
   <property name="text">
    <string>Books Database</string>
   </property>
  </action>
  <action name="actionQuotes_Database">
   <property name="text">
    <string>Quotes Database</string>
   </property>
  </action>
  <action name="actionMovies_Database">
   <property name="text">
    <string>Movies Database</string>
   </property>
  </action>
  <action name="actionTags_Database">
   <property name="text">
    <string>Tags Database</string>
   </property>
  </action>
  <action name="actionClients_Database">
   <property name="text">
    <string>Clients Database</string>
   </property>
  </action>
  <action name="actionBooks">
   <property name="text">
    <string>Books</string>
   </property>
  </action>
  <action name="actionMovies">
   <property name="text">
    <string>Movies</string>
   </property>
  </action>
  <action name="actionQuotes">
   <property name="text">
    <string>Quotes</string>
   </property>
  </action>
  <action name="actionTags">
   <property name="text">
    <string>Tags</string>
   </property>
  </action>
  <action name="actionClients">
   <property name="text">
    <string>Clients</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>PlatformServerGUI</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
